base_user_show_email/views/res_users_view.xml

```diff
@@ -20,7 +20,7 @@
         </field>
     </record>
     <record id="user_email_tree" model="ir.ui.view">
-        <field name="name">res.users.show_email.tree</field>
+        <field name="name">res.users.show_email.list</field>
```

Comment:
Not blocking, but this is an unnecessary change that can break extensions inheriting it.